modules/etcd-defrag.adoc

```diff
@@ -6,10 +6,43 @@
 [id="etcd-defrag_{context}"]
 = Defragmenting etcd data
 
-Manual defragmentation must be performed periodically to reclaim disk space after etcd history compaction and other events cause disk fragmentation.
+Defragmentation of etcd data reclaims disk space after etcd history compaction and other events  cause disk fragmentation.
 
 History compaction is performed automatically every five minutes and leaves gaps in the back-end database. This fragmented space is available for use by etcd, but is not available to the host file system. You must defragment etcd to make this space available to the host file system.
 
+Defragmentation occurs using one of the following options:
+* Automatic
+* Manual
+
+[NOTE]
+====
+Automatic defragmentation is preferred because the etcd operator uses cluster information to determine the most efficient operation for the user.
```

Comment:
ISG recommends limiting "is preferred/is recommended/etc." in favor of stronger language. 

I guess one way to go about this here would be to state that automated defrags are good for most cases, but you might want to rely on manual for... reasons. Not sure what they'd be.